roiextractors/extractors/suite2p/suite2psegmentationextractor.py

```diff
@@ -67,8 +69,53 @@ def roi_locations(self):
         return np.array([j['med'] for j in self.stat]).T
 
     @staticmethod
-    def write_segmentation(segmentation_extractor, nwb_file_path):
-        return NotImplementedError
+    def write_segmentation(segmentation_object, save_path):
+        save_path = Path(save_path)
+        if not save_path.is_dir():
+            raise ValueError('provide a folder as save_path')
+        if segmentation_object.__class__.__name__=='MultiSegmentationExtractor':
```

Comment:
Is there a reason we can't use `isinstance` here?

Reply:
no, this should be `isinstance` :)

Reply:
Basically, try to avoid any cases where you are programming against the string representations of objects. This was the same reason I asked you to get rid of string matching against the names of attributes. It's not so much a performance problem as a style problem and added complexity to the code. Things like automatic refactoring will catch one and not the other. Also, if you are comparing a child of a class, `isinstance` will work and comparing the name won't.

That said, there are cases where this could be useful, for instance when you are not able to import the necessary class (which is the case in the next line, because we can't use the class in the definition of that class)

Reply:
okay, good to know